setup.py

```diff
@@ -54,7 +54,7 @@
       tests_require=tests_require,
       extras_require=dict(tests=tests_require,
                           zip_export=['ftw.zipexport'],
-                          ldap=['python-ldap', 'Products.PloneLDAP']),
+                          ldap=['python-ldap < 3.0.0', 'Products.PloneLDAP']),
```

Comment:
Yes, `3.0.0b1 < 3.0.0` is actually true: `setuptools` has a specific version comparison implementation.
So you want to change that to `python-ldap < 3.0.0b1`

Reply:
Ok, I'm using `3.0.0b1` now.